### PR TITLE
Support for VersionPress

### PR DESCRIPTION
**Pull request by pavelevap**
Related: #4341 

Work in progress...

### CODE REVIEW

File: .versionpress/schema.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+pods_relationship:
+  table: podsrel
+  id: id
+  references:
+    pod_id: post
+    field_id: post
+    item_id: post
```
Comment (sc0ttkclark) [inline .versionpress/schema.yml]: The complexity of what Pods does makes this a little trickier, the item ID could be related to a post, a term, a user, etc depending on the pod the field was added on.

File: .versionpress/schema.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    related_pod_id: post
+    related_field_id: post
+    related_item_id: post
```
Comment (sc0ttkclark) [inline .versionpress/schema.yml]: The complexity of what Pods does makes this a little trickier, the item ID could be related to a post, a term, a user, etc depending on the object the relationship is related to.

File: .versionpress/schema.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+postmeta:
+  value-references:
+    meta_key@meta_value:
+      _pods_*[/\d+/]: post
```
Comment (sc0ttkclark) [inline .versionpress/schema.yml]: The complexity of what Pods does makes this a little trickier, the item ID could be related to a post, a term, a user, etc depending on the object the relationship is related to.

File: .versionpress/schema.yml (hunk 1 above)
Comment (pavelevap) [inline .versionpress/schema.yml]: Right, thank you! I did not realize that, I quickly started with extended post. We will have to use custom function, similar example is used in VP core for `_menu_item_object_id` ([here](https://github.com/versionpress/versionpress/blob/d3d0c984dff174d70ca67555a0b0b9ca325958f3/plugins/versionpress/.versionpress/schema.yml#L30)). Pods API will probably have similar function to get item type for postmeta value? I will try to look at it.

**Comment by Ramoonus** [comment]
travis hangs on PHP 5.3
not really an issue imo

**Comment by sc0ttkclark** [comment]
More work is needed here, just noting the link for their docs https://docs.versionpress.net/en/developer/plugin-support/

**Comment by sc0ttkclark** [comment]
We can revisit this in the future if VersionPress ends up becoming active again

**Pull request closed without merging**